guide/using-visual-cpp-with-webots.md

```diff
@@ -134,3 +134,31 @@ the `Pause` button then the `Revert` button. Then, in Visual C++, use the `Debug
 process. Still in Visual C++, you can now add breakpoints and watches in the
 controller code. Then, in Webots, hit the `Run` button to resume the simulation.
 Now the controller should pause when it reaches one of your breakpoints.
+
+
+### Link with the Webots libraries
+
+Webots contains several `C` or `C++` libraries based on the `libController` or
+`libCppController` libraries (e.g. the `automobile library`, `DARwIn-OP library`,
+etc.).
+The corresponding precompiled libraries are incompatible with Visual C++,
+because they are compiled using the `gcc` compiler tool chain embedded in Webots.
+
+To link these libraries with your project, their source files should be built directly in your project,
+exactly as the `libCppController` is linked (cf. instructions above).
+
```

Comment:
This sentence is not very clear, because we don't link these libraries (it's not possible). Rather we re-compile them...